content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -236,19 +236,18 @@ with `NoExecute` effect. If the fault condition returns to normal the kubelet or
 controller can remove the relevant taint(s).
 
 {{< note >}}
-To maintain the existing [rate limiting](/docs/concepts/architecture/nodes/)
-behavior of pod evictions due to node problems, the system actually adds the taints
-in a rate-limited way. This prevents massive pod evictions in scenarios such
-as the master becoming partitioned from the nodes.
+The control plane limits the rate of adding node new taints to nodes. This rate limiting
+manages the number of evictions that are triggered when many nodes become unreachable at
+once (for example: if there is a network disruption).
 {{< /note >}}
 
-The feature, in combination with `tolerationSeconds`, allows a pod
-to specify how long it should stay bound to a node that has one or both of these problems.
+You can specify `tolerationSeconds` for a Pod to define how long that Pod should stay bound
+to a Node that has problems.
```

Comment:
@sftim, I am just noting content that could use future edits or cleanup.
possible edit:
You can specify `tolerationSeconds` for a Pod to define how long that Pod stays bound
to a failing [or unresponsive] Node.